src/board.py

```diff
@@ -39,3 +40,10 @@ def set_cells_with_value(self, points: list[Point], value: bool) -> None:
                 raise ValueError(f"{point} out of bounds!")
 
             self.__cells[point.x][point.y] = value
+
+    def is_on_bottom(self, figure: Figure) -> bool:
```

Comment:
This method returns nothing (`None`) after `for`

Reply:
Tnx! Done